controllers/apply/under10k/__snapshots__/form.test.js.snap

```diff
@@ -20,6 +20,8 @@ Array [
   "organisationStartDate: Enter a month and year",
   "organisationAddress: Enter a full UK address",
   "organisationType: Select a type of organisation",
+  "accountingYearDate: Enter a day and month",
```

Comment:
This is the result of moving the logic up a level. It essentially means that until until you answer the `organisationStartDate` question we assume you are going to need to answer these questions. This is arguably more intentional behaviour. Affects the summary screen:

**Before**

![image](https://user-images.githubusercontent.com/123386/79881137-c2ad2580-83e8-11ea-84cd-4dae4ba0cd13.png)

**After**

![image](https://user-images.githubusercontent.com/123386/79881020-a27d6680-83e8-11ea-80c6-903409e31b6a.png)

This doesn't change the end result. But a judgement call here, new behaviour is to allow you to fill in these fields and strip them only once you've told us the date, vs. assuming they are not needed to begin with. Feels more honest as the new organisation rule is the exception rather than the default.

If we want to retain the old behaviour we should update the `isNewOrganisation` check to return `true` if a start date hasn't been provided yet. It defaults to `false` right now.